### ISSUE TITLE
Focus is lost when pressing "Hide" with screen reader (violation)

### ISSUE THREAD

**Issue by Maren-SSB** [PxTools/pxtools-project#30]
VoiceOver (skjermleser) leser opp «ingen handling tilgjengelig, knapp» etter trykk på «Skjul»-knappen i navigation drawer. Det som egentlig skal skje, er det samme som skjer ved tastaturnavigering, og det er at fokus flyttes tilbake til elementet som åpnet innholdet (f.eks. Filter-knappen eller Lagre-knappen).

Vurdering fra Useit: Dette kan skyldes at fokus ikke returneres til trigger-elementet (f.eks. “Lagre”) når innholdet lukkes. Da havner fokus i et udefinert område, noe som gir utydelige tilbakemeldinger fra skjermleser.

Dette er et brudd på 3.2.2, siden fokus forsvinner og en havner et sted som ikke gir mening.

<br>

## Akseptansekriterier

- Ved trykk på «Skjul» med skjermleser skal fokus flyttes tilbake til elementet som åpnet innholdet
